velox/exec/tests/TableScanTest.cpp

```diff
@@ -622,6 +622,36 @@ TEST_F(TableScanTest, count) {
   EXPECT_EQ(numRead, 10'000);
 }
 
+TEST_F(TableScanTest, batchSize) {
+  // One row is larger than 1KB. Total input is less than 10MB.
+  auto numColumns = 2 * 1024 / sizeof(int64_t);
+  auto numRows = 1024;
+
+  std::vector<std::string> names;
+  for (int i = 0; i < numColumns; i++) {
+    names.push_back(fmt::format("c{}", i));
+  }
+  auto rowType =
+      ROW(std::move(names), std::vector<TypePtr>(numColumns, BIGINT()));
+  auto vector = makeVectors(1, numRows, rowType);
+
+  auto filePath = TempFilePath::create();
+  writeToFile(filePath->path, vector);
+
+  createDuckDbTable(vector);
+
+  auto task = AssertQueryBuilder(duckDbQueryRunner_)
+                  .plan(PlanBuilder().tableScan(rowType).planNode())
+                  .splits(makeHiveConnectorSplits({filePath}))
+                  .assertResults("SELECT * FROM tmp");
+  const auto opStats = task->taskStats().pipelineStats[0].operatorStats[0];
+
+  SCOPED_TRACE(fmt::format(
```

Comment:
Should this be an assertion, i.e. shouldn't we assert that opStats.rawInputBytes) < 10MB? But, then, why does this matter?